routes/post_events/post_events.js

```diff
@@ -0,0 +1,32 @@
+// endpoints related to post events
+const express = require('express');
+const pool = require('../../db');
+
+const postEventsRouter = express();
+
+postEventsRouter.use(express.json());
+// endpoints related to post events
+
+function snakeToCamel(postevents) {
+  return postevents.map((postevent) => ({
+    posteventId: postevent.postevent_id,
+    eventId: postevent.event_id,
+    description: postevent.description,
+  }));
+}
+
+// Update Post Event Endpoint
+postEventsRouter.post('/:id', async (req, res) => {
+  try {
+    const { description } = req.body;
+    const updatePostEvents = await pool.query(
+      'UPDATE postevent SET description =$1 WHERE postevent_id = $2 RETURNING *;',
+      [description, req.params.id],
+    );
+    res.json(snakeToCamel(updatePostEvents.rows));
```

Comment:
please include error handling for the following cases:

- query failed, no data returned
- successful query, data returned (current line of code)

please also add status code 200